README.md

```diff
@@ -80,6 +80,12 @@ There is already [PythonInMusic](https://wiki.python.org/moin/PythonInMusic) but
 * [SIDEKIT](http://lium.univ-lemans.fr/sidekit/)³ - Speaker and Language recognition
 * [SpeechRecognition](https://github.com/Uberi/speech_recognition) -  Wrapper for several ASR engines and APIs, online and offline
 * [talkbox](http://scikits.appspot.com/talkbox)² - General speech/signal processing algorithms. ⚠️ Not maintained
+* [aeneas](https://www.readbeyond.it/aeneas/) - Forced aligner, based on MFCC+DTW, 35+ languages
+* [FAVE-align](https://github.com/JoFrhwld/FAVE/) - Forced aligner, based on HTK (HMM), English. ⚠️ Not on PyPI
```

Comment:
FAVE seems [not to be actively developed anymore](https://github.com/JoFrhwld/FAVE/graphs/contributors). Is this still being used a lot? Otherwise I would propose to remove this from the list.